CRM/Import/Parser.php

```diff
@@ -1484,4 +1468,25 @@ protected function removeEmptyValues($array) {
     return $array;
   }
 
+  /**
+   * Given an array of contact values, figure out the contact type.
+   *
+   * @param array $values
+   * @return string
+   */
+  protected function guessContactType(array $values): string {
```

Comment:
I think this can got in `ImportParser` (which is part of civiimport & not shared with the contact import)